src/libs/brand-utils.js

```diff
@@ -0,0 +1,28 @@
+import { brands } from 'src/libs/brands'
+import { getConfig } from 'src/libs/config'
+import * as Utils from 'src/libs/utils'
+
+
+// TODO: Deprecate Datastage (https://broadworkbench.atlassian.net/browse/SATURN-1414)
```

Comment:
Can you move the comment down to the line of code it relates to?

Reply:
I was thinking it'd be harder to miss up top but I don't feel strongly about it. Will move it.

Reply:
Thanks, I was just confused because it had no relationship to the following line.

Reply:
Good to know and a reason enough to move it for me 😊

Reply:
Changed in [9d11924](https://github.com/DataBiosphere/terra-ui/pull/3257/commits/9d11924c69e01df4057dd2b51b770eb6f3e1c02e).